app/api/api_v1/routers/search.py

```diff
@@ -68,9 +69,14 @@ def _search_request(
         if use_vespa:
             data_access_search_params = create_vespa_search_params(db, search_body)
             # TODO: we may wish to cache responses to improve pagination performance
-            data_access_search_response = _VESPA_CONNECTION.search(
-                parameters=data_access_search_params
-            )
+            try:
+                data_access_search_response = _VESPA_CONNECTION.search(
+                    parameters=data_access_search_params
+                )
+            except QueryError:
+                raise HTTPException(
```

Comment:
Do we want to log this here as well?

Reply:
Maybe, there is already some logging done as [part of the data access library](https://github.com/climatepolicyradar/data-access/blob/main/src/cpr_data_access/search_adaptors.py#L102-L107).  Do you have thoughts on wether we need anything further?